package.json

```diff
@@ -1,86 +1,65 @@
 {
   "name": "r7insight_node",
-  "version": "1.8.4",
-  "description": "Logentries client for node; use with or without Winston or Bunyan.",
+  "version": "2.0.0",
+  "description": "Rapid7 Insight Platform client for node; use with or without Winston or Bunyan.",
   "keywords": [
     "logentries",
     "logentries.com",
     "logs",
     "logging",
     "winston",
-    "bunyan"
+    "bunyan",
+    "insightops",
+    "rapid7",
+    "rapid7.com",
+    "insightplatform",
+    "insight"
   ],
   "homepage": "https://github.com/rapid7/r7insight_node",
   "author": {
-    "name": "Darien Maillet Valentine",
-    "email": "darien.valentine@logentries.com"
+    "name": "Rapid7",
+    "email": "InsightOpsTeam@rapid7.com"
   },
-  "contributors": [
-    {
-      "name": "Darien Maillet Valentine",
-      "email": "darien.valentine@logentries.com"
-    },
-    {
-      "name": "Paul Dixon",
-      "email": "paul.dixon@logentries.com"
-    }
-  ],
   "license": "BSD-3-Clause",
   "repository": {
     "type": "git",
     "url": "https://github.com/rapid7/r7insight_node.git"
   },
-  "main": "lib/logger.js",
+  "main": "src/index.js",
   "scripts": {
-    "build": "gobble build lib --force",
     "lint": "eslint src",
-    "prepublish": "npm run build",
-    "test": "npm run build && npm run lint && node test/test.js | tap-spec"
+    "lint:fix": "eslint src --fix",
+    "test": "npm run lint && node test/test.js | tap-spec",
+    "test:debug": "node test/test.js | tap-spec"
   },
   "engines": {
-    "iojs": ">=0.10",
-    "node": ">=0.8.0",
-    "npm": ">=1.4.6"
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
```

Comment:
This doesn't match with the Travis config

Reply:
In what way? missing `8.10.0`?

Reply:
travis is missing 8, yes, but also why do we allow only 10.13.* in the 10.x release? any specific reason for it?

Reply:
oh, yes, this line is the lowest supported node version from all the packages we use, this exact line appears in that packages `package.json`